recipes/boost/all/test_package/data.txt

```diff
@@ -1,2 +1,2 @@
 123
-234
\ No newline at end of file
+234
```

Comment:
changing test data is a bit risky (I did not check if the test would still pass or not, but the hook should not warn on such files)

Reply:
I don't think boost is using this file.